charts/incubator/redisinsight/values.yaml

```diff
@@ -1,36 +1,40 @@
 image:
   pullPolicy: IfNotPresent
   repository: tccr.io/truecharts/redisinsight
-  tag: latest@sha256:723214d13e55e0a65f7f698e4f431ffe2e2da8e5e859c89c86040ffc672cfafc
+  tag: latest@sha256:6e56b7490ac8b4cb2d11522d494484bfd00dd20108853cda0ac241861ccb42e6
```

Comment:
can we change latest to v1.14.0
```yaml
image:
  pullPolicy: IfNotPresent
  repository: tccr.io/truecharts/redisinsight
  tag: v1.14.0@sha256:569d8cfd3352ef17800d0c1cb3437ce4510f49571c1d61bbb3422ebc1fd1bb0e
```

Reply:
So the issue we use the DockerHub image -> https://hub.docker.com/r/redislabs/redisinsight/tags

But upstream the packages are using different versions-> https://github.com/RedisInsight/RedisInsight

I'm down with changing it since it's incubator and I don't know the difference, but as long as I documented it here might as well be known